src/stylesheets/blocks/_footer.scss

```diff
@@ -29,7 +29,7 @@
     
     .lg-text {
       font-size: rem(14);
-      line-height: rem(24);
+      line-height: rem(26);
```

Comment:
unitless is recommended which is something I just do and had never looked up why but apparently here is the why: https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0

Reply:
thanks for catching this. i'm actually using `em` most places and forgot to here. sounds like that might not be the way to go either?

Reply:
yeah, unitless is probably the way to go...so just `line-height: 1.5` or whatever...just no units